### PR TITLE
Fix elsa configuration

### PR DESCRIPTION
**Pull request by cameronmarsh**

### CODE REVIEW

File: apps/andi/lib/andi/application.ex [hunk 1]
```diff
@@ -5,6 +5,21 @@ defmodule Andi.Application do
...
+        true ->
+          [{"localhost", 9092}]
+
+        false ->
+          kafka_brokers
+          |> String.split(",")
+          |> Enum.map(&String.trim/1)
+          |> Enum.map(fn entry -> String.split(entry, ":") end)
+          |> Enum.map(fn [host, port] -> {String.to_atom(host), String.to_integer(port)} end)
+      end
```
Comment (LtChae) [inline apps/andi/lib/andi/application.ex:21]: Why was this removed from the config?

**Review by LtChae — APPROVED** [review]

File: apps/andi/lib/andi/application.ex (hunk 1 above)
Comment (cameronmarsh) [reply apps/andi/lib/andi/application.ex:21]: yeah that was a mistake
